Zend/zend_execute.c

```diff
@@ -3127,6 +3127,9 @@ static zend_always_inline void zend_fetch_property_address(zval *result, zval *c
 		}
 	}
 
+	/* Pointer on property callback is required */
+	ZEND_ASSERT(zobj->handlers->get_property_ptr_ptr != NULL);
```

Comment:
I don't object for clarity, but for debugging purposes this doesn't seem meaningfully different. Would it make more sense to move this to object construction? That would've caught it even without a corresponding test that triggers the property access.

Reply:
I see, ok will try.